ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/BackingStore.hs

```diff
@@ -62,6 +62,9 @@ data BackingStore m keys values diff = BackingStore {
   , bsValueHandle :: !(m (WithOrigin SlotNo, BackingStoreValueHandle m keys values))
     -- | Apply a valid diff to the contents of the backing store
   , bsWrite       :: !(SlotNo -> diff -> m ())
+    -- | Retrieve statistics: the sequence number, and the number of entries
```

Comment:
I'm just wondering if we want to use a generic name such as `stats` or if we want to make this more descriptive of what these stats actually are. I have no strong opinion.